oca/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
```

Comment:
@lasley This file **must** only contain `__path__ = __import__('pkgutil').extend_path(__path__, __name__)` to keep oca as a reusable namespace package.
https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages

Reply:
I think the complexity of having the package split doesn't pay. It's just easier to have all in one place, this shouldn't grow up so much as to be unmaintainable. Just a collection of helpers...

Reply:
@Yajo I think we must keep the possibility to release others python packages under the oca namespace. (ie: utilities to use in our dev/release process, ????).  There is nothing complex here.